From aed8d3573463d92bd11309b74504a065c7affe8a Mon Sep 17 00:00:00 2001
From: Ziad Elhanafy <ziad.elhanafy@arm.com>
Date: Mon, 6 Nov 2023 10:31:33 +0000
Subject: [PATCH] Add zephyr module file

Add zephyr module file to include CMakeLists.txt and Kconfig
located in the Arm Automotive Solutions repository.

Upstream-Status: Inappropriate [Arm Automotive Solutions-specific]
Signed-off-by: Ziad Elhanafy <ziad.elhanafy@arm.com>
---
 zephyr/module.yml | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 zephyr/module.yml

diff --git a/zephyr/module.yml b/zephyr/module.yml
new file mode 100644
index 0000000..ec0704d
--- /dev/null
+++ b/zephyr/module.yml
@@ -0,0 +1,5 @@
+name: psa-arch-tests
+
+build:
+  cmake-ext: True
+  kconfig-ext: True
-- 
2.25.1

